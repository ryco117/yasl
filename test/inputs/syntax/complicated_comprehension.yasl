const a = [ 10, 2, 45, 6 ]

echo 's' ~ [ b for b <- a ]->join(', ') ~ [ b->tostr() for b <- a]->join(' ') ~ ' asdasd'
