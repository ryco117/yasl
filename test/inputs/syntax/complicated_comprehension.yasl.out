s10, 2, 45, 610 2 45 6 asdasd
